drivers/storage/scaleio/scaleio.go

```diff
@@ -686,6 +791,14 @@ func (d *driver) storagePoolName() string {
 	return d.r.Config.GetString("scaleio.storagePoolName")
 }
 
+func (d *driver) thinOrThick() string {
+	thinOrThick := d.r.Config.GetString("scaleio.thinOrThick")
+	if thinOrThick == "" {
+		return "ThinProvisioned"
```

Comment:
Hi @clintonskitson,

I'd suggest this as an alternative:

```go
type ScaleioProvisionType int

const (
    Thin ScaleioProvisionType = iota
    Thick
)
```

This avoids free-text in many places, other than parsing it from the CLI.